### PR TITLE
foliate: init at 2.6.3

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Addresses issue https://github.com/NixOS/nixpkgs/issues/108461

### CODE REVIEW

**Comment by veprbl** [comment]
@GrahamcOfBorg eval

File: pkgs/applications/office/foliate/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  src = fetchFromGitHub {
+    owner = "johnfactotum";
+    repo = pname;
+    rev = version;
+    sha256 = "0ribqaxl8g1i83fxbn288afwbzzls48ni57xqi07d19p9ka892mr";
+  };
+
+  nativeBuildInputs =
+    [ meson ninja python3 wrapGAppsHook ];
```
Comment (SuperSandro2000) [inline pkgs/applications/office/foliate/default.nix:17]:
```suggestion
  nativeBuildInputs = [ meson ninja python3 wrapGAppsHook ];
```

File: pkgs/applications/office/foliate/default.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    webkitgtk
+    desktop-file-utils
+    gobject-introspection
+    gsettings-desktop-schemas
+  ];
+
+  meta = with lib; {
+    description = "A simple and modern GTK eBook reader";
+    homepage = "https://johnfactotum.github.io/foliate/";
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/office/foliate/default.nix:43]: Missing maintainer and gpl3Only or gpl3Plus?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/office/foliate/default.nix [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  nativeBuildInputs =
+    [ meson ninja python3 wrapGAppsHook ];
+
+  postPatch = ''
+    patchShebangs build-aux/meson/postinstall.py
+  '';
+
+  postFixup = ''
+    echo "fixing wrapper"
+    sed -i "1 a imports.package._findEffectiveEntryPointName = () => 'com.github.johnfactotum.Foliate';" $out/bin/.com.github.johnfactotum.Foliate-wrapped
```
Comment (SuperSandro2000) [inline pkgs/applications/office/foliate/default.nix:25]:
```suggestion
    sed -i "1 a imports.package._findEffectiveEntryPointName = () => 'com.github.johnfactotum.Foliate';" $out/bin/.com.github.johnfactotum.Foliate-wrapped
```

File: pkgs/applications/office/foliate/default.nix [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  };
+
+  nativeBuildInputs = [ meson ninja python3 wrapGAppsHook ];
+
+  postPatch = ''
+    patchShebangs build-aux/meson/postinstall.py
+  '';
+
+  postFixup = ''
+    echo "fixing wrapper"
```
Comment (AndersonTorres) [inline pkgs/applications/office/foliate/default.nix:23]: It is not good to pollute the stdout with those messages.

I love Gutenberg debugging, but it should not to be in production code.

Comment (onny) [reply pkgs/applications/office/foliate/default.nix:23]: Sorry for the late fix ;D https://github.com/NixOS/nixpkgs/pull/167977

**Pull request merged**